tests/API/XCCDF/guide/Makefile.am

```diff
@@ -11,4 +11,5 @@ EXTRA_DIST = \
 	all.sh \
 	content-xccdf11.xml \
 	content-xccdf12.xml \
-	content-value-xccdf11.xml
+	content-value-xccdf11.xml \
+	benchmark_id_test-ds.xml
```

Comment:
`benchmark_id_test-ds.xml` was not added to this commit, and caused the jenkins fail